airbyte-integrations/connectors/source-harness/metadata.yaml

```diff
@@ -4,17 +4,17 @@ data:
       - api.harness.io
   remoteRegistries:
     pypi:
-      enabled: true
+      enabled: false
       packageName: airbyte-source-harness
   registryOverrides:
     oss:
-      enabled: false
```

Comment:
Wait it was disabled? weird. I think that is partially because the CI says it fails.